src/bijectors/corr.jl

```diff
@@ -321,7 +325,7 @@ function _link_chol_lkj(W::UpperTriangular)
     return z
 end
 
-_link_chol_lkj(W::LowerTriangular) = _link_chol_lkj(transpose(W))
+_link_chol_lkj_from_lower(W::AbstractMatrix) = _link_chol_lkj_from_upper(transpose_eager(W))
```

Comment:
This one is a bit weird because I just saw that it has it's own chainrule defined while the evaluation itself does _not_ have a specialized implementation..